org.aliflang.lang.yml

```diff
@@ -0,0 +1,149 @@
+app-id: org.aliflang.lang
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: alifstudio
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
```

Comment:
```suggestion
```
you have this in filesystem=host already

Reply:
you are right